zio-kafka/src/main/scala/zio/kafka/consumer/Consumer.scala

```diff
@@ -111,13 +143,27 @@ trait Consumer {
    * On completion of the stream, the consumer is unsubscribed. In case of multiple subscriptions, the total consumer
    * subscription is changed to exclude this subscription.
    */
+
   def plainStream[R, K, V](
     subscription: Subscription,
     keyDeserializer: Deserializer[R, K],
     valueDeserializer: Deserializer[R, V],
     bufferSize: Int = 4
   ): ZStream[R, Throwable, CommittableRecord[K, V]]
 
+  /**
+   * Like [[plainStream]] but wraps the stream in a construct that ensures graceful shutdown
+   */
```

Comment:
Since this method might be the most attractive way to use zio-kafka, lets extend the documentation a bit.

e.g.
```suggestion
  /**
   * Like [[plainStream]] but wraps the stream in a construct that ensures graceful shutdown. During a graceful shutdown the consumer is stopped but the stream can complete processing and commit already fetched records.
   *
   * Example usage:
   * {{{
   *   ... todo ...
   * }}}
   */
```

We could also include a reference to the documentation (though I am always extremely happy when the scaladocs are all you need, e.g. scalatest documentation is my benchmark).